src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
Because default is well..the default so when the selection handler tried to select the 'best' handler, it comes back with our OG one regardless of entity type. This is why creating a user entity is currently fataling too. See `SelectionPluginManager::getPluginId` and `SelectionPluginManager::getSelectionGroups`